### PR TITLE
Inline docs for modules/holiday-snow.php

### PR DESCRIPTION
**Pull request by aheckler**

### CODE REVIEW

File: modules/holiday-snow.php [hunk 1]
```diff
@@ -31,16 +31,44 @@ public function blog_field_html() {
...
 		do_action( 'jetpack_holiday_snow_option_updated' );
 	}
 }
 
 function jetpack_holiday_snow_script() {
+	
+	/**
+	 * Filter the holiday snow.
+	 * 
+	 * There's no actual "chance" for snow, it's either on or off.
```
Comment (jeherve) [inline modules/holiday-snow.php]: I would disagree. defining this filter to return `true` doesn't mean that there will be snow falling on your site. You still have to enable the snow option in Settings > Reading in your dashboard.

Comment (aheckler) [inline modules/holiday-snow.php]: That's technically correct I suppose, but inline documentation is documentation *of the code*, not the user interface we have in wp-admin. That type of documentation belongs on Jetpack.me or in wp-admin itself.

Even you've done inline docs this way as well. ;) For example, your PR to add inline docs to Related Posts includes this docblock:

https://github.com/Automattic/jetpack/blob/add/inline-docs-related-posts/modules/related-posts/jetpack-related-posts.php#L1138-L1144

To quote you, with a few simple changes:

> defining this filter to return `true` doesn't mean that there will be [related posts] on your site. You still have to enable the [related post module] in [Jetpack > Settings] in your dashboard.

If I wanted to take it even further, I could say "Well, why doesn't every filter and action have note saying that Jetpack needs to be enabled for it to work?" 

IMHO, inline docs should only be documenting the filter/action that follows, not every related setting that appears anywhere else. ¯\_(ツ)_/¯

Comment (jeherve) [inline modules/holiday-snow.php]: That's true, but we wouldn't want a dev to think that by setting that filter to `true`, they'd force the snow to appear regardless of every other setting or date format.

How about "Can the snow be displayed?"

**Comment by aheckler** [comment]
Cleared up the wording. @jeherve: thoughts?

**Comment by jeherve** [comment]
@aheckler Looks great!

**Pull request merged**